cmd/helm/search.go

```diff
@@ -60,14 +64,62 @@ func newSearchCmd(out io.Writer) *cobra.Command {
 	}
 
 	f := cmd.Flags()
-	f.BoolVarP(&o.regexp, "regexp", "r", false, "use regular expressions for searching")
-	f.BoolVarP(&o.versions, "versions", "l", false, "show the long listing, with each version of each chart on its own line")
-	f.StringVar(&o.version, "version", "", "search using semantic versioning constraints")
+	f.StringVar(&o.searchEndpoint, "endpoint", "https://hub.helm.sh", "monocular instance to query for charts")
```

Comment:
Looking at the flag documentation, do you think it would be better to have a `helm search repositories|repos` and `helm search hub` for each of the search types with `helm search` being an alias for `helm search hub`? I think it might be less confusing than using a flag to switch it

Reply:
If we're following convention from `helm get`, it's probably best to make `helm search` display usage help for the subcommands `helm search hub` and `helm search repositories`. That way the two commands can accept different flags, different arguments, etc and users won't be confused about using, say, why a flag like `--index-file` won't work with `helm search hub` but works with `helm search repositories`.

Reply:
That would also help in the future when we support OCI registries; it'd just be another subcommand: `helm search registry`

Reply:
Yeah, I agree with this.

Reply:
@bacongobbler it's worth noting that `helm get` does something by default. It just doesn't display information. :)

In this case I will have it display the subcommand information.